CHANGELOG.md

```diff
@@ -0,0 +1,13 @@
+# Changelog
+
+## Unreleased Changes
+- N/A
+
+## v1.2.0
+- Reverted the breaking behavior change in v1.1.0 that broke custom authentication flows
+
+## v1.1.0 (Not reccomended for use, this functionality will be ported to the 2.0 update)
```

Comment:
why mention 1.1.0 at all? why burden users with groking this knowledge that they dont need

in the same spirit, why v1.2.0?

at most should we just saw `## v1.1.0 (unreleased)`